### ISSUE TITLE
ref) CNN - LeNet5 

### ISSUE THREAD

**Issue by LeeGitaek** [LeeGitaek/Detection-Using-Convolutional-Networks#2]
http://yann.lecun.com/exdb/publis/pdf/lecun-01a.pdf

My LeNet5 implementation link : https://github.com/LeeGitaek/2020_AI_Class/blob/master/MNIST_CNN_LeNet5.ipynb

<img width="766" alt="스크린샷 2020-07-01 오전 4 25 49" src="https://user-images.githubusercontent.com/5070020/86168389-1f4c5f80-bb53-11ea-8112-df0c608ec43e.png">

    class LeNet5(torch.nn.Module):
         def __init__(self):
         super(LeNet5,self).__init__()

        self.c1 = torch.nn.Conv2d(1,6,kernel_size=5,stride=1,padding=0)
        self.x1 = torch.nn.Tanh()
        self.s2 = torch.nn.AvgPool2d(kernel_size=2,stride=2)

        self.c3 = torch.nn.Conv2d(6,16,kernel_size=5,stride=1,padding=0)
        self.x3 = torch.nn.Tanh()
        self.s4 = torch.nn.AvgPool2d(kernel_size=2,stride=2)

        #self.fl = torch.nn.Faltten()

        self.f5 = torch.nn.Linear(16*5*5,120,bias=True)
        self.x5 = torch.nn.Tanh()

        self.f6 = torch.nn.Linear(120,84,bias=True)
        self.x6 = torch.nn.Tanh()

        self.output = torch.nn.Linear(84,10,bias=True)

       def forward(self,x):
        out = self.c1(x)
        out = self.x1(out)
        out = self.s2(out)
        out = self.c3(out)
        out = self.x3(out)
        out = self.s4(out)
        out = out.view(out.size(0),-1)
        out = self.f5(out)
        out = self.x5(out)
        out = self.f6(out)
        out = self.x6(out)
        out = self.output(out)
        return out
